airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CDKVersionTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.mysql;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.github.zafarkhaja.semver.Version;
+import io.airbyte.cdk.CDKConstants;
```

Comment:
@cpdeethree - For our discussion later - this import is currently failing.

Reply:
Update: @edgao very helpfully tried to repro locally on his machine and it ran successfully on his machine. With that as guidance I tried again after nuking my local caches, and now it is looking good! 🎉 

Thanks again @edgao. 🙏